core/messages/en.json

```diff
@@ -215,7 +215,7 @@
       "shippingMethodLabel": "Shipping method",
       "summaryTotalLabel": "Total",
       "destinationTitle": "Destination",
-      "destinationWithCountTitle": "Destination {number}/{total}",
+      "destinationWithCountTitle": "Destination {number, number}/{total, number}",
```

Comment:
We were passing in numbers instead of strings so I added these for the ICU format. If the intent was to pass strings, we could update the call site instead.